### ISSUE TITLE
WCAG 2.1 Conformance Statement for Portals link does not work

### ISSUE THREAD

**Issue by monemansari** [MicrosoftDocs/powerapps-docs#2982]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b8e40af7-ba15-42f8-b9e1-3d1c5ae73285
* Version Independent ID: d63fa884-aa15-e95d-f44b-8ce6eabf7142
* Content: [Accessibility in Power Apps portals - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/portals/admin/accessibility)
* Content Source: [powerapps-docs/maker/portals/admin/accessibility.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/maker/portals/admin/accessibility.md)
* Service: **powerapps**
* Sub-service: **portals**
* GitHub Login: @neerajnandwana-msft
* Microsoft Alias: **nenandw**

**Comment by monemansari** [comment]
Find here https://cloudblogs.microsoft.com/industry-blog/government/2018/09/11/accessibility-conformance-reports/
